modules/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageClientSettings.java

```diff
@@ -120,6 +120,17 @@ public class GoogleCloudStorageClientSettings {
         () -> PROXY_HOST_SETTING
     );
 
+    /**
+     * The maximum number of retries to use when a GCS request fails.
+     * <p>
+     * Default to 5 to match {@link com.google.cloud.ServiceOptions#getDefaultRetrySettings()}
+     */
+    static final Setting.AffixSetting<Integer> MAX_RETRIES_SETTING = Setting.affixKeySetting(
+        PREFIX,
+        "max_retries",
+        (key) -> Setting.intSetting(key, 5, 0, Setting.Property.NodeScope)
+    );
```

Comment:
We never configured number of retries for GCS previously. The default settings were for 6 attempts (aka 5 retries)